guides/plugins/plugins/api/custom-pricing.md

```diff
@@ -0,0 +1,96 @@
+# Custom Pricing
+
+The Custom Pricing feature allows massive advances in the pricing model capabilities in the Shopware 6 ecosystem.
+
+The API interface exposed by this module allows the user to operate a set of commands which will enable the granular 
+''overriding'' of prices via an external data repository or ERP system. This is achieved by defining a custom 
+relationship between a current price and one of the following entities Customer.
+
+### Pre-requisites and setup
+
+As Custom Pricing is part of the Commercial plugin, it requires an existing Shopware 6 installation and the activated
+Shopware 6 Commercial plugin on top. This plugin Commercial plugin can be installed as per the familiar
+[install instructions](guides/plugins/plugins/plugin-base-guide#install-your-plugin). In addition, Custom Prices need 
+to be activated within your merchant's account.
+
+## Working with the API route
+
+To create, alter and delete customer-specific prices, you can use the API endpoint `/api/_action/product-customer-price`. As like with any other admin request in Shopware, you first need to take care of authenticating yourself. Therefore, please head over to the 
+[authentication guide](https://shopware.stoplight.io/docs/admin-api/ZG9jOjEwODA3NjQx-authentication) for details.
+
+Apart from that, the Custom Pricing API interface models itself upon the interface of the sync API, so you will 
+be able to package your requests similarly, see our [API documentation](https://shopware.stoplight.io/docs/admin-api).
+
+{% hint style="info" %}
+You can use the route with single `upsert` and `delete` actions or even combine those in a single request. This way, you can pack several different commands inside one sync API request, and each of them would be executed independent and isolate
+{% endhint %}
+
+So, it's not surprising that the request body looks like a familiar sync request. In the payload for the `upsert` action, you pass the following data:
+* `productId`: The product whose price should be overwritten
+* `customerId`: The customer for whom we'd like to have a custom price for
+* `price`: The new custom price you want to use. It's defined by how you're familiar when working with prices in Shopware.
```

Comment:
```suggestion
* `price`: The new custom price you want to use.
```
Not actually sure this is correct, we have included the quantityStart and quantityEnd within the payload, and then there are sub-prices with the same schema as the normal price json from `product_price`